### PR TITLE
check for understanding part e

### PR DESCRIPTION
**Pull request by Kamilahsantos**
In this PR I convert check for understandings into questions, of issue #1007 

check for understanding at datetime, basic-js-practice and runtime-complexity files

### CODE REVIEW

File: datetime/datetime.md [hunk 1]
```diff
@@ -58,4 +58,12 @@ Instructor demonstrates in the video walk through how to work with Date Object i
...
\ No newline at end of file
+- What is the Date Object?
+
+- What day corresponds to 0 in Date()?
+
+- What month corresponds to 0 in Date()?
+
+- What are the advantages of using numerical dates (milliseconds) vs. using object dates (`new Date()`) ? Talk with a partner
+
+- How to add and subtract dates?
```
Comment (alodahl) [inline datetime/datetime.md]:
```suggestion
- How do you add and subtract dates?
```

File: javascript/basic-js-practice.md [hunk 2]
```diff
@@ -178,3 +178,17 @@ common([1, 2, 3], [1, 3]) → True
 ### [Challenge 10] Determine Runtime Complexity
 
 At the top your document for each challenge, state the runtime complexity for that function.
+
+
+### Check for Understanding
+
+- How create a function in Javascript?
```
Comment (alodahl) [inline javascript/basic-js-practice.md]:
```suggestion
- Demonstrate how to create a function in Javascript.
```

File: javascript/basic-js-practice.md [hunk 3]
```diff
@@ -178,3 +178,17 @@ common([1, 2, 3], [1, 3]) → True
...
+
+- Create 3 examples using Math()
```
Comment (alodahl) [inline javascript/basic-js-practice.md]:
```suggestion
- Create 3 examples using Math().
```

File: javascript/basic-js-practice.md [hunk 4]
```diff
@@ -178,3 +178,17 @@ common([1, 2, 3], [1, 3]) → True
...
+
+- How to return a function
+
+- Write a loop 
+
+- Create a object
```
Comment (alodahl) [inline javascript/basic-js-practice.md]:
```suggestion
- Create an object.
```

File: javascript/basic-js-practice.md [hunk 5]
```diff
@@ -178,3 +178,17 @@ common([1, 2, 3], [1, 3]) → True
...
 At the top your document for each challenge, state the runtime complexity for that function.
+
+
+### Check for Understanding
+
+- How create a function in Javascript?
+
+- Create 3 examples using Math()
+
+- How to return a function
```
Comment (alodahl) [inline javascript/basic-js-practice.md]: What do you mean here? ‘How does a function return its output’? Or ‘How do you call a function’?

File: javascript/basic-js-practice.md [hunk 6]
```diff
@@ -178,3 +178,17 @@ common([1, 2, 3], [1, 3]) → True
...
+
+- Write a loop
```
Comment (alodahl) [inline javascript/basic-js-practice.md]: Demonstrate how to write a loop.

File: runtime-complexity/runtime-complexity.md [hunk 7]
```diff
@@ -108,4 +108,10 @@ Discuss with the group and an instructor:
 
 ### Check for Understanding
 
-Make a cheat sheet about runtime complexity. For O(1), O(log(n)), O(n), and O(nlogn) and O(n^2), give an example of 1-3 algorithms/operations that have this runtime.
+- How important is it to understand runtime complexity?
```
Comment (alodahl) [inline runtime-complexity/runtime-complexity.md]:
```suggestion
- Why is it important to understand runtime complexity?
```

File: runtime-complexity/runtime-complexity.md [hunk 8]
```diff
@@ -108,4 +108,10 @@ Discuss with the group and an instructor:
...
+
+- What is Big-O/runtime?
+
+- What others names to Runtime complexity  that are generally used interchangeably?
```
Comment (alodahl) [inline runtime-complexity/runtime-complexity.md]: Can you reword this? Not sure what this means

File: datetime/datetime.md (hunk 1 above)
Comment (Kamilahsantos) [reply datetime/datetime.md]: thanks, i commit the suggestion

File: javascript/basic-js-practice.md (hunk 2 above)
Comment (Kamilahsantos) [reply javascript/basic-js-practice.md]: thanks, i commit the suggestion

File: javascript/basic-js-practice.md (hunk 3 above)
Comment (Kamilahsantos) [reply javascript/basic-js-practice.md]: thanks, i commit the suggestion

File: javascript/basic-js-practice.md (hunk 4 above)
Comment (Kamilahsantos) [reply javascript/basic-js-practice.md]: thanks, i commit the suggestion

File: javascript/basic-js-practice.md (hunk 5 above)
Comment (Kamilahsantos) [reply javascript/basic-js-practice.md]: I think in "How does a function return its output" , propose an exercise to fix how to "return" the function

Comment (alodahl) [reply javascript/basic-js-practice.md]: ok, can you ask them to demonstrate it?

**Pull request merged**